sensor/kubernetes/complianceoperator/updater_test.go

```diff
@@ -49,6 +51,7 @@ func (s *UpdaterTestSuite) SetupSuite() {
 }
 
 func (s *UpdaterTestSuite) SetupTest() {
+	centralcaps.Set([]centralsensor.CentralCapability{centralsensor.ComplianceV2Integrations})
```

Comment:
I wonder if this could cause some side effect in other tests running in the same package? `centralcaps` is a global variable

Reply:
This is the only test and the `updater` is the only thing in this package that uses `centralcaps`.  I'm open to better ideas if we have examples of dealing with these things in other ways.

Reply:
I suggest cleaning up the centralcaps in the `TeardownTest` function. If any other tests are added in this package they could have side-effects depending on the order they run.